src/mono/sample/Android/AndroidSampleApp.csproj

```diff
@@ -79,6 +79,7 @@
         Assemblies="@(BundleAssemblies)"
         MainLibraryFileName="$(AssemblyName).dll"
         StripDebugSymbols="$(StripDebugSymbols)"
+        StaticLinkedRuntime="true"
```

Comment:
```suggestion
```
I don't think we should do this just to temporarily bump the APK size metric back down.  We understand the underlying issues (diagnostics component was enabled by default; symbols were not stripped from shared libraries).

The android sample should use dynamic components by default to mimic what the real android workload would do.

Reply:
I suggested doing this before I realized https://github.com/dotnet/runtime/pull/53240 would help.  If we can land that PR , then I think this change won't be needed and we can add size validation tests separately.  

@lateralusX @directhex make sense to you?

Reply:
Sure, we can keep dynamic linking as long as we are aware size will be affected until #53240 gets done.

Reply:
I would also like that to strip components shared libraries, or they will be adding a lot to size as well when included.